components/Layout/index.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import Link from 'next/link';
+import { Layout, Menu, Icon, Typography, Avatar, Breadcrumb } from 'antd';
+import './index.scss';
+
+const { Header, Sider, Content, Footer } = Layout;
+const { Title, Text } = Typography;
+
+const CustomLayout = props => (
+	<React.Fragment>
+		<Layout className="layout" hasSider>
+			<Sider
+				className="sider"
+				breakpoint="md"
+				collapsedWidth={0}
+				onBreakpoint={broken => {
+					console.log(broken);
+				}}
+				onCollapse={(collapsed, type) => {
+					console.log(collapsed, type);
+				}}
+			>
+				<Title level={2} className="sider__logo">
+					{' '}
+					Leer{' '}
+				</Title>
+				<div className="sider__profile">
+					<Avatar size={80} icon="user" />
+					<Text className="sider__profile-text text-white">
+						Hello Jude!
+					</Text>
+				</div>
+
+				<nav className="sider__menu">
+					<Menu
+						theme="dark"
+						mode="inline"
+						defaultSelectedKeys={[props.selectedMenuItem]}
+					>
+						<Menu.Item key="home">
```

Comment:
map over an array of item details to create the `Menu.Item`s. When ever you see that you are writing the same piece of code over and over again, you are doing something wrong

Reply:
Okay, Thanks. Fixed this in my latest commit